scripts/build/Platform/Windows/build_config.json

```diff
@@ -582,5 +583,33 @@
     "PARAMETERS": {
       "CDK_VERSION": "2.68.0"
     }
+  },
+  "install_mono_android_api_24_release": {
+    "TAGS": [],
+    "PIPELINE_ENV": {
+      "NODE_LABEL":"windows-packaging"
```

Comment:
`windows-packaging` is meant to be used for codesigning executables for installers/distributables. Is that what we're making from the SDK builds?

Otherwise, the standard `windows-latest` or labels used for android builds should be used

Reply:
This is meant for the windows installer, we would need the same android setup for windows-packaging as we do with windows-latest.